src/client/runner/automation/playback/select/select-editable-content.js

```diff
@@ -0,0 +1,65 @@
+import testCafeCore from '../../../deps/testcafe-core';
+import * as selectUtils from './utils';
+import SelectBaseAutomation from './base';
+
+var textSelection   = testCafeCore.textSelection;
+var contentEditable = testCafeCore.contentEditable;
+var positionUtils   = testCafeCore.positionUtils;
+
+
+export default class SelectEditableContentAutomation extends SelectBaseAutomation {
+    constructor (startNode, endNode) {
+        super(contentEditable.getNearestCommonAncestor(startNode, endNode));
+
+        var startOffset = contentEditable.getFirstVisiblePosition(startNode);
+        var endOffset   = contentEditable.getLastVisiblePosition(endNode);
+
+        var startPos      = { node: startNode, offset: startOffset };
+        var endPos        = { node: endNode, offset: endOffset };
+        var startPosition = contentEditable.calculatePositionByNodeAndOffset(this.element, startPos);
+        var endPosition   = contentEditable.calculatePositionByNodeAndOffset(this.element, endPos);
+
+        if (startPosition > endPosition) {
+            startOffset = contentEditable.getLastVisiblePosition(startNode);
+            endOffset   = contentEditable.getFirstVisiblePosition(endNode);
+        }
+
+        // NOTE: We should recalculate nodes and offsets for selection because we
+        // may have to select children of expectedStartNode and expectedEndNode
+        startPos = contentEditable.calculateNodeAndOffsetByPosition(startNode, startOffset);
+        endPos   = contentEditable.calculateNodeAndOffsetByPosition(endNode, endOffset);
+
+        this.startNode   = startPos.node;
+        this.startOffset = startPos.offset;
+        this.endNode     = endPos.node;
+        this.endOffset   = endPos.offset;
+    }
+
+    _calculateAbsolutePoint (isStart) {
```

Comment:
(startNode, startOffset) and (endNode, endOffset). you little mixed ut up.
But I think we don't really need in _calculateAbsolutePoint